packages/apps/src/SideBar/index.tsx

```diff
@@ -17,13 +17,10 @@ import translate from '../translate';
 import Item from './Item';
 
 type Props = I18nProps & {
-  children?: React.ReactNode
+  children?: React.ReactNode;
```

Comment:
Generally inside `{}` (although both forms are acceptable from a checking pov), we tend to use `,`